src/main/java/org/jboss/xavier/integrations/route/MainRouteBuilder.java

```diff
@@ -70,6 +70,9 @@
 
     private List<Integer> httpSuccessCodes = Arrays.asList(HttpStatus.SC_OK, HttpStatus.SC_CREATED, HttpStatus.SC_ACCEPTED, HttpStatus.SC_NO_CONTENT);
 
+    @Value("${minio.host:x}")
+    private String minio_host;
```

Comment:
Maybe a comment to remember that is is just for testing environment

Reply:
I have moved it to the test class.